asteroid/filterbanks/enc_dec.py

```diff
@@ -253,7 +251,7 @@ def forward(self, spec):
                 padding=self.padding,
                 output_padding=self.output_padding,
             )
-        elif spec.ndim > 3:
+        else:
```

Comment:
do you have an alternative to this? this is an important fix for torch.script as it does not allow forward passes that would have `Optional[torch.Tensor]`

Reply:
I don't exactly understand the question. 
This looks like it will always return a torch.Tensor with the `else` right?

Reply:
yes, and this is required for torch.script as a path without a returned tensor is not allowed

Reply:
the questions is if setting this path to the default is okay for yiou

Reply:
Yes, that seems alright. 
I don't think people will give 1-dim tensor as spectrograms..